src/main/java/org/opensearch/neuralsearch/processor/chunker/IFieldChunker.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.neuralsearch.processor.chunker;
+
+import java.util.Map;
+import java.util.List;
+
+public interface IFieldChunker {
```

Comment:
again, add java docs for interface and it's APIs so that other people know what they are for and how to use them.

Reply:
Already added